signature/src/mechanism/simple/mod.rs

```diff
@@ -102,13 +185,12 @@ pub fn judge_single_signature(
     Ok(())
 }
 
-#[allow(unused_assignments)]
-pub fn get_signatures(image: &mut image::Image) -> Result<Vec<Vec<u8>>> {
+pub async fn get_signatures(image: &mut Image) -> Result<Vec<Vec<u8>>> {
```

Comment:
Yes, I surely agree with you. IMO, from `http/https://`, ... and some other protocols can be supported, maybe something like https://github.com/sigstore/cosign/blob/890cec1f43a8ec0754e0dd5a5d120847b63b6c4e/pkg/signature/keys.go#L44
However it will be left in the future, right?

Reply:
Yes, thanks.